### ISSUE TITLE
KubeWise chat message can have optional extra text (like the k8s cluster)

### ISSUE THREAD

**Issue by cep21** [larderdev/kubewise#5]
I want to install KubeWise for multiple k8s clusters (staging and production).  I would like to have the message it sends clearly show which cluster the helm command is running in.  Maybe optional text I can append to the text it runs, so I can say "staging: " or "production: " at the start.

**Comment by dtuite** [comment]
I like this idea a lot.

My personal preference is to create separate slack channels for staging and production and use `--set slack.channel="#staging"` to direct the messages to each channel. That way you can have different settings for each channel. You might want notifications silenced in the #staging channel for example.

However, I can see how there will be situations where you want them both going to the same channel. Maybe you have low install volume and don't want to bother with separate channels.

The current implementation looks [like this in the Slack message playground](https://api.slack.com/docs/messages/builder?msg=%7B%22text%22%3A%22%F0%9F%92%BD%20Installing%20*zookeeper*%20version%20*1.3.3*%20into%20namespace%20*zookeeper*%20via%20Helm.%20%E2%8F%B3%5Cn%5Cn%60%60%60These%20are%20the%20install%20notes%20from%20Helm%60%60%60%22%7D).

![current slack message design](https://user-images.githubusercontent.com/562403/76972506-95bbad80-6926-11ea-9f24-7ae4aa3718d8.png)

Perhaps if the user passed

     --set slack.messagePrefix="`production`\n"

then they could get a message which looks [like this](https://api.slack.com/docs/messages/builder?msg=%7B%22text%22%3A%22%60production%60%5Cn%20%F0%9F%92%BD%20Installing%20*zookeeper*%20version%20*1.3.3*%20into%20namespace%20*zookeeper*%20via%20Helm.%20%E2%8F%B3%5Cn%5Cn%60%60%60These%20are%20the%20install%20notes%20from%20Helm%60%60%60%22%7D):

![new slack message design](https://user-images.githubusercontent.com/562403/76972335-5ee59780-6926-11ea-8314-d1a09990d89c.png)

What do you think?

**Comment by cep21** [comment]
That's fine with me, I'm not picky :)  My alternative would be to make multiple kubewise apps: call one of them "KubeWise-Staging" and another "KubeWise-Production".

**Comment by dtuite** [comment]
@cep21 this feature is added and released in version `0.5.0`.

```
helm install kubewise -n kubewise larder/kubewise --set messagePrefix="\`production\` " --set handler=slack --set slack.token="<api-token>" --version 0.5.0
```
